spec/unit/unparser_spec.rb

```diff
@@ -267,6 +267,16 @@ def foo(bar:, baz: "value")
           assert_source '{ :-@ => 1 }'
         end
       end
+
+      context 'method' do
```

Comment:
I think that we should identify the "root" cause of the crash and create a dedicated test case. Just putting in a "random" S-Expression that lead to an unparser crash before your fix is not adequately securing the code base from regressions.

I'll look into this now.